packages/compass-preferences-model/src/global-config.ts

```diff
@@ -91,11 +91,11 @@ async function loadGlobalPreferences(
 const cliProps = Object.entries(allPreferencesProps).filter(
   ([, definition]) => definition.cli
 );
-function getCliPropNamesByType(type: AmpersandType<any>): string[] {
+function getCliPropNamesByType(type: unknown): string[] {
```

Comment:
This type could be `Types` from Joi, I think?

```suggestion
function getCliPropNamesByType(type: Types): string[] {
```